### ISSUE TITLE
Issues with codeblocks

### ISSUE THREAD

**Issue by tonyknight** [0xreeko/gpt2markdown#12]
This is a great extension and works really well. Just what I was looking for.

I did notice one issue that cause my codeblocks to be malformed. The [```<span>python] needs to have a  linebreak before it render properly. I was able to fix this with find/replace. Just add a linebreak before this tag, and it looks bueno...

Thanks,
Tony
![Screenshot 2023-03-16 at 11 03 15](https://user-images.githubusercontent.com/265037/225711984-a7de2182-7479-48ab-bb3f-f693a1d10fdc.jpg)
![Screenshot 2023-03-16 at 11 03 36](https://user-images.githubusercontent.com/265037/225712052-028ad3bf-99c4-41a4-a661-b6838fd92842.jpg)

**Comment by banool** [comment]
Yeah I found this too, some markdown renderers need there to be a newline prior to the codeblock.
